modules/core/src/main/java/org/apache/ignite/internal/processors/pool/PoolProcessor.java

```diff
@@ -150,6 +151,9 @@ public Executor poolForPolicy(byte plc) throws IgniteCheckedException {
 
                 return ctx.getSchemaExecutorService();
 
+            case GridIoPolicy.CALLER_THREAD:
```

Comment:
Please, describe in comments what is caller thread io policy?

Reply:
It's a part of CommunicationIo, I don't think we should comment it here.

Reply:
The comment should be on enum value. This mean that IO thread invokes a listener itself instead of pushing it to any executor. It helps to manage the execution place and context smarter.

Reply:
May be rename it to IO_THREAD?

Reply:
`org.apache.ignite.internal.processors.closure.GridClosureProcessor#callLocalSafe(java.util.concurrent.Callable<R>, byte)` - it uses policies as well but it isn't an IO part.